### ISSUE TITLE
Telegram shareUrl changed

### ISSUE THREAD

**Issue by proArtex** [tabalinas/jssocials#195]
According to this https://telegram.me/share/url?url=urlOfTheWebsite&text=TextToAccompany shareUrl for telegram was changed and it is already doesn't work on desktop apps. Now it is tg://msg_url?url={url}&text={text}

**Comment by Ruud68** [comment]
Hi,
am I correct when saying that what you want is an additional telegram icon: for the desktop / browser?
a similar change I did for facebook messenger here: https://github.com/tabalinas/jssocials/pull/166 ?

so you have 'telegram' > pointing to tg://...
and
'telegram_browser' > pointing to telegram.me/share...

**Comment by proArtex** [comment]
@Ruud68 nope, it is about default shareUrl for telegram. The link above is from telegram doc, so check the url from "share" btn: tg://msg_url?url=urlOfTheWebsite&text=TextToAccompany. Now it is: tg://msg?text={url} {text}

**Comment by Ruud68** [comment]
A okay, I see. Thanks for clearifying.
I can do the PR for that

**Comment by Ruud68** [comment]
Hi, the URL change is part of this PR: https://github.com/tabalinas/jssocials/pull/196
@tabalinas needs to review / implement :)
